tests/test_bigbio.py

```diff
@@ -100,22 +100,35 @@ def runTest(self):
         (10) test_coref_ids: Check if text matches offsets in coreferences
 
         """  # noqa
+        self.setUp()
```

Comment:
this will run `setUp` twice right? https://docs.python.org/3/library/unittest.html#organizing-tests

Reply:
Yes, nice catch!

Reply:
Should be fixed by fcba9d8